_posts/2017-11-08-cost-of-drupal-7.md

```diff
@@ -0,0 +1,180 @@
+
+#### Easy Drupal upgrades forever
+
+![A happy Drupal sunrise]({{ site.url }}/assets/img/blog/the-promise-of-making-drupal-upgrades-easy-742x1114.jpg)<br>
+  <span class="caption"> Image from Dries's blog post </span>
+
+To the surprise of the community, in an abrupt departure from business-as-usual in early 2017, Dries committed to ["easy upgrades forever"](https://dri.es/making-drupal-upgrades-easy-forever), starting with Drupal 8 of course. The short of it is Drupal 8 to 9 upgrades should be **far easier** (and less expensive) than any previous major version upgrade, and so will be from here on in. That means for those not yet on Drupal 8, you only have one final difficult upgrade left in your Drupal journey until the end of time.
+
+This is a fairly natural outcome given the possibilities afforded by a more structured, object-oriented architecture coupled with the growing desire to ease upgrade pain that has been building for some time. Although the difficult, technical work remains to [flesh out exactly how this will be done](https://www.drupal.org/core/d8-bc-policy), the commitment from the top is worth putting a lot of stock in, and the community is well underway in putting this grand proclamation into technical reality. When upgrades are far easier, they will help rectify some of the sentiment of leaving the smaller sites behind, since major version upgrades will be a much less daunting task here on in.
```

Comment:
> Although the difficult, technical work remains to flesh out exactly how this will be done, the commitment from the top is worth putting a lot of stock in, and the community is well underway in putting this grand proclamation into technical reality.

Some repeated words and unneeded commas, change to:

> Although difficult technical work is needed to flesh out exactly how this will be done, the commitment from the top is worth putting stock in and the community is on the way to making this grand proclamation a reality.